From 1f145832e58ded081513df661a67eaf0d9faf669 Mon Sep 17 00:00:00 2001
From: Heinrich Toews <heinrich.toews@wago.com>
Date: Thu, 14 Dec 2017 17:45:36 +0000
Subject: [PATCH 135/261] dts: pfc: rtc-rs5c372: add rtc osc adjustment only
 for old PFCs

        IMPORTANT: We skip setting osc adjustment for G2 devices.
---
 arch/arm/boot/dts/wago-pfc100.dtsi   | 1 +
 arch/arm/boot/dts/wago-pfc200.dtsi   | 1 +
 arch/arm/boot/dts/wago-pfc200v2.dtsi | 1 +
 arch/arm/boot/dts/wago-src.dtsi      | 1 +
 4 files changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/wago-pfc100.dtsi b/arch/arm/boot/dts/wago-pfc100.dtsi
index a071aa7..705f2f0 100644
--- a/arch/arm/boot/dts/wago-pfc100.dtsi
+++ b/arch/arm/boot/dts/wago-pfc100.dtsi
@@ -596,6 +596,7 @@
 
 		interrupt-parent = <&gpio3>;
 		interrupts = <21 IRQ_TYPE_LEVEL_LOW>;
+                trim-data = <0 0 13>;
         };
 
 
diff --git a/arch/arm/boot/dts/wago-pfc200.dtsi b/arch/arm/boot/dts/wago-pfc200.dtsi
index e91b92b..c944d07 100644
--- a/arch/arm/boot/dts/wago-pfc200.dtsi
+++ b/arch/arm/boot/dts/wago-pfc200.dtsi
@@ -535,6 +535,7 @@
 
 		interrupt-parent = <&gpio1>;
 		interrupts = <1 IRQ_TYPE_LEVEL_LOW>;
+                trim-data = <0 0 13>;
         };
 
 	24c512@54 {
diff --git a/arch/arm/boot/dts/wago-pfc200v2.dtsi b/arch/arm/boot/dts/wago-pfc200v2.dtsi
index 43f0180..fcaa71f 100644
--- a/arch/arm/boot/dts/wago-pfc200v2.dtsi
+++ b/arch/arm/boot/dts/wago-pfc200v2.dtsi
@@ -690,6 +690,7 @@
 
 		interrupt-parent = <&gpio3>;
 		interrupts = <21 IRQ_TYPE_LEVEL_LOW>;
+                trim-data = <0 0 13>;
         };
 
         tps: tps65218@24 {
diff --git a/arch/arm/boot/dts/wago-src.dtsi b/arch/arm/boot/dts/wago-src.dtsi
index 0d95148..5af8d10 100644
--- a/arch/arm/boot/dts/wago-src.dtsi
+++ b/arch/arm/boot/dts/wago-src.dtsi
@@ -582,6 +582,7 @@
 
 		interrupt-parent = <&gpio3>;
 		interrupts = <21 IRQ_TYPE_LEVEL_LOW>;
+                trim-data = <0 0 13>;
 	};
 
 	/* CARD-Port-Expander */
-- 
2.7.4

